Tests/XCUITests/SearchTest.swift

```diff
@@ -255,4 +269,22 @@ class SearchTests: BaseTestCase {
             XCTAssert(keyboardsCount > 0, "The keyboard is not shown")
         }
     }
+
+    func testOpenTabsInSearchSuggestions() {
```

Comment:
Please remember when a test has Functional, Smoke & Sanity on Test Rail sub suite, it should run only as part of the SmokeTest. I see this SearchTest suite part of the Smoketest4, it should be enabled there.
It is a good practice also to add // Smoketest before the declaration of the test, L272

Reply:
Will do, need to make a habit out of this.

Reply:
Thanks! it takes time to get used to the process